packages/blockchain-wallet-v4-frontend/src/data/components/nfts/sagas.ts

```diff
@@ -72,6 +73,54 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas; network
     }
   }
 
+  const fetchNftUserPreferences = function* () {
+    try {
+      const prefs = S.getNftUserPreferences(yield select())
+      if (Remote.Success.is(prefs)) return
+      yield put(A.fetchNftUserPreferencesLoading())
+      const retailToken = yield call(generateRetailToken)
+      const res: ReturnType<typeof api.getNftUserPreferences> = yield call(
+        api.getNftUserPreferences,
+        retailToken
+      )
+
+      // first time user, opt-out
```

Comment:
opt-in right?

Reply:
opt-out

Reply:
yeah comment is kinda confusing, system is opt-out